README.md

```diff
@@ -620,6 +620,20 @@ They are based on these older repositories:
 
 - [EF Core Power Tools](https://marketplace.visualstudio.com/items?itemName=ErikEJ.EFCorePowerTools&ssr=false#overview) - Visual Studio extension that can generate an Entity Framework Core DbContext and model classes, and various diagrams directly from your MSBuild.Sdk.SqlProj project.
 
+## .NET runtime support
+
+To use this SDK you need to have a .NET runtime installed. Depending on the SDK version you use, varying versions of the .NET runtime (released once a year) is required.
```

Comment:
Technically I believe we support specific .NET SDK versions which come with a bundled runtime version. So MSBuild.Sdk.SqlProj version 3.0.0 supports .NET SDK 8.0 and 9.0. I'm not sure that is entirely clear from this.

Reply:
Any suggestions for rephrasing?

Reply:
```suggestion
MSBuild.Sdk.SqlProj requires a .NET SDK to be installed. Depending on the version of MSBuild.Sdk.SqlProj you can use varying versions of the .NET SDK (released once a year). The following table lists the supported .NET SDK versions by each MSBuild.Sdk.SqlProj version:
```